librecad/src/src.pro

```diff
@@ -285,6 +285,7 @@ SOURCES += \
     lib/gui/rs_graphicview.cpp \
     lib/gui/rs_grid.cpp \
     lib/gui/rs_linetypepattern.cpp \
+    lib/gui/rs_linetypepattern.cpp \
```

Comment:
accidentally changed?
This is wrong and could be avoided when using `git diff` before committing.